c7n/resources/apigw.py

```diff
@@ -1229,3 +1229,22 @@ def generate_arn(self):
             )
 
         return self._generate_arn
+
+
+@resources.register("apigwv2-stage")
```

Comment:
The tags for this resource is coming back as `'Tags': {'string': 'string'}`

Reply:
fixed, thanks for looking!